### PR TITLE
Benchmark L0_perf_nomodel

### PR DESCRIPTION
**Pull request by CoderHam**

### CODE REVIEW

File: qa/L0_perf_nomodel/run_test.sh [hunk 1]
```diff
@@ -25,15 +25,7 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
```
Comment (deadeyegoodwin) [inline qa/L0_perf_nomodel/run_test.sh:28]: Why remove this? It is in all our CI tests.

Comment (CoderHam) [reply qa/L0_perf_nomodel/run_test.sh:28]: I believe this check already happens in our test.sh file and so I believe it was no longer in run_test.sh file. Should we still have it in the run_test.sh as it is always run from the test file itself?

Comment (deadeyegoodwin) [reply qa/L0_perf_nomodel/run_test.sh:28]: You are right, keep it like it is.

Comment (CoderHam) [reply qa/L0_perf_nomodel/run_test.sh:28]: @deadeyegoodwin should I make this change across the other perf run_test files?

Comment (deadeyegoodwin) [reply qa/L0_perf_nomodel/run_test.sh:28]: I don't think they use the REPO_VERSION directly for this others... but you can check.

File: qa/L0_perf_nomodel/test.sh [hunk 2]
```diff
@@ -79,26 +81,34 @@ TEST_NAMES=(
     "${UNDERTEST_NAME} 16MB I/O Latency HTTP"
     "${UNDERTEST_NAME} Maximum Throughput GRPC"
     "${UNDERTEST_NAME} Maximum Throughput HTTP")
```
Comment (deadeyegoodwin) [inline qa/L0_perf_nomodel/test.sh:83]: Here an other places below you left the closing paren here instead of moving it to the end

Comment (CoderHam) [reply qa/L0_perf_nomodel/test.sh]: Fixed

**Pull request merged**